libsql-server/tests/embedded_replica/mod.rs

```diff
@@ -452,6 +456,7 @@ fn replica_no_resync_on_restart() {
 
 #[test]
 fn replicate_with_snapshots() {
+    const ROW_COUNT: i64 = 120;
```

Comment:
This would be a question for @MarinPostma : I had to change number of inserts in this tests from 200&rarr;120. When passing over 155 inserts on my local machine, the test fails as it says that the simulation cannot finish within 10sec.

The thing is that I've checked and these 150+ requests finish within ~600ms. I've checked if generated streams don't leak and it looks like they are properly dropped, causing stream close on the server side as well.

I'm still trying to find out why 200 inserts loop causes a problem.

Reply:
You can instead increase the simulation duration on the builder, just set it to an arbitrary duration

Reply:
How is the simulation time being computed? My problem is that I'm hitting 10s simulation execution limit, yet the real time of the test is <1s:
![image](https://github.com/tursodatabase/libsql/assets/679705/3c413f8d-1295-454c-a48e-f2487fc0ac70)

Reply:
Time is ticked very fast in simultatins, but time doesn't tick at the same speed in blocking threads, just set "simulation_duration" in the builder to something large enough